src/signals/incident/containers/KtoContainer/index.js

```diff
@@ -199,8 +198,9 @@ export const KtoContainer = () => {
               </header>
             ) : (
               <StyledHeading>
-                {isSatisfied ? 'Ja, ik ben' : 'Nee, ik ben niet'} tevreden met
-                de behandeling van mijn melding
+                {isSatisfied
```

Comment:
moet hier nog een test van worden gemaakt? komt de regel voor in de coverage?

Reply:
Komt voor in de coverage en de huidige test voldoet nog.